src/app/components/history/history.component.html

```diff
@@ -1,19 +1,36 @@
 <div class="container-fluid">
+  <div class="row mb-2">
+    <div class="btn-group btn-group-toggle">
+      <label class="btn btn-secondary" for="filter_all" [class.active]="filter === 'all'">
+        <input type="radio" [ngModel]="filter" value="all" name="options" id="filter_all" (ngModelChange)="filterHistory($event)" autocomplete="off"> {{ 'All' | translate }}
+      </label>
+      <label class="btn btn-secondary" for="filter_delegated" [class.active]="filter === 'delegated'">
+        <input type="radio" [ngModel]="filter" value="delegated" name="filter" id="filter_delegated" (ngModelChange)="filterHistory($event)" autocomplete="off"> {{ 'Delegated' | translate }}
+      </label>
+      <label class="btn btn-secondary" for="filter_undelegated" [class.active]="filter === 'undelegated'">
+        <input type="radio" [ngModel]="filter" value="undelegated" name="filter" id="filter_undelegated" (ngModelChange)="filterHistory($event)" autocomplete="off"> {{ 'Undelegated' | translate }}
```

Comment:
Change from `{{ 'Undelegated' | translate }}` to `{{ 'UNDELEGATED_PLURAL' | translate }}`